app/views/registrations/login.html.erb

```diff
@@ -0,0 +1,29 @@
+<p class="text-center h1 fw-bold mb-5 mx-1 mx-md-4 mt-4">Login</p>
```

Comment:
@rishabh-vinsol 
currently if a user signup but deleted the verify account email by mistake.
How can he login or get the mail again?